opensearch-k8s-operator.yaml

```diff
@@ -0,0 +1,50 @@
+package:
+  name: opensearch-k8s-operator 
+  version: 2.6.1
+  epoch: 0
+  description: OpenSearch Kubernetes Operator
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - ca-certificates-bundle
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - git
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/opensearch-project/opensearch-k8s-operator 
+      tag: v${{package.version}}
+      expected-commit: b2417013b570cc2dec3acd8204c15f73ba887efa 
+
+  - uses: go/bump
+    with:
+      modroot: ./opensearch-operator
+      deps: golang.org/x/net@v0.23.0 google.golang.org/protobuf@v1.33.0
+
+  - uses: go/build
+    with:
+      packages: ./opensearch-operator
+      output: manager
+      ldflags: -s -w
+
+  - uses: strip
```

Comment:
given -w is the default in go/build pipeline, we can omit this as well.

Reply:
Was I supposed to drop ` - uses: strip`?